paper-button-behavior.html

```diff
@@ -14,35 +14,42 @@
 <script>
 
   /** @polymerBehavior */
+  Polymer.PaperButtonBehaviorImpl = {
+
+    properties: {
+
+      _elevation: {
+        type: Number
+      }
+
+    },
+
+    observers: [
+      '_calculateElevation(focused, disabled, active, pressed, receivedFocusFromKeyboard)'
+    ],
+
+    hostAttributes: {
+      role: 'button',
+      tabindex: '0'
+    },
+
+    _calculateElevation: function() {
+      var e = 1;
+      if (this.disabled) { // Note: or !raised
```

Comment:
Ah, nevermind, I read `paper-button` and understood. The comment is a little weird. Maybe `Note: or !raised, when used by <paper-button>` ?